Class1.java

```diff
@@ -0,0 +1,59 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+​
+public class Class1 {
+    public final String SPECIAL_CHARACTER = "a";
+​
+    public List<String> method(String paragraph) {
```

Comment:
change method name to something more descriptive of what the method does(i.e. find shortestWordInEachSentence)

Reply:
also get as input the character u wanna check for so that it is more general.